netci.groovy

```diff
@@ -1927,7 +1927,7 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                         case 'innerloop':
                         case 'no_tiered_compilation_innerloop':
                             if (configuration == 'Checked') {
-                                isDefaultTrigger = true
+                                isDefaultTrigger = false
```

Comment:
`isDefaultTrigger = false` is the default, so just remove the conditions.

Reply:
I left the innerloop jobs on purpose - if we find an issue with AzDev we would have a way to trigger the arm innerloop jobs. I will remove them as soon as the AzDev pipeline goes through enough testing

Reply:
> Is this change intended to move arm32/arm64 default triggered PR jobs from Jenkins to AzDev?

Not only, we are also enabling other jobs in AzDev against PRs but keeping them in Jenkins to monitor consistency between the corresponding legs